UnitTests/StringCalculatorUnitTests.cs

```diff
@@ -44,5 +44,11 @@ public int GivenInputHasLineBreaksOrCommasSeparatedNumbersWhenAddThenReturnsSumO
             return _stringCalculator.Add(input);
         }
         
+        [Test]
+        public void GivenInputHasNegativeNumberWhenAddThenReturnsAnException()
+        {
+            Exception ex = Assert.Throws<System.ArgumentException>(delegate() { _stringCalculator.Add("-1,2,-3");});
```

Comment:
`Assert.Throws<System.ArgumentException>(delegate() { _stringCalculator.Add("-1,2,-3");});`

You can simplify this further by using a Lambda expression like this
`Assert.Throws<System.ArgumentException>(() => _stringCalculator.Add("-1,2,-3"));`